### ISSUE TITLE
After school class

### ISSUE THREAD

**Issue by benjaminlees** [foundersandcoders/london-community-outreach#27]
I would like to engage a local school to see if there is interest in an after school beginners class for front end development. My hope is that this goes well and we can create a full on FAC course for school leavers.

I am totally new to this discussion and therefor have a bunch of questions.

- Are we in contact with any schools , if so which ones?
- Has this all been discussed already?

**Comment by sofer** [comment]
@benjaminlees I agree that we should be aiming for a FAC course for school leavers. We have made a small start in that direction. It's going to be great to get you involved.

> Are we in contact with any schools , if so which ones?

Yes, we are in contact with [Mulberry UTC](https://github.com/foundersandcoders/london-programme/issues/127) in a big way and we are currently doing workshops attended by pupils of Mulberry, Morpeth, Green Springs, and Wapping High. See https://github.com/foundersandcoders/spotlight-workshop

> Has this all been discussed already?

No. The discussions have only just started. See https://github.com/foundersandcoders/london-community-outreach/issues
